cava/Cava2HDL/Makefile

```diff
@@ -20,6 +20,9 @@ all:		haskell
 
 haskell:
 		cabal build
+		runhaskell Setup.hs configure --user
+		runhaskell Setup.hs build
+		runhaskell Setup.hs install
```

Comment:
I don't think Setup.hs is required? Were you seeing issues without it?

Reply:
With the cabal setup, installing is not require. `cabal repl` should work directly to get a `ghci` prompt with the files loaded.

Reply:
Well, I don't want to use `cabal repl`! OK... I am going to do your bidding and remove these lines and use `cabal repl` :-(